---a/net/minecraft/inventory/ICrafting.java
+++b/net/minecraft/inventory/ICrafting.java
@@ -1,8 +1,10 @@
 package net.minecraft.inventory;
 
 import java.util.List;
+
 import net.minecraft.item.ItemStack;
 
-public interface ICrafting
-{
+public interface ICrafting {
+
     void sendContainerAndContentsToPlayer(Container containerToSend, List itemsList);
 
     void sendSlotContents(Container containerToSend, int slotInd, ItemStack stack);
@@ -12,4 +12,4 @@
     void sendProgressBarUpdate(Container containerIn, int varToUpdate, int newValue);
 
     void func_175173_a(Container p_175173_1_, IInventory p_175173_2_);
-}
+}
